webui/Readme.md

```diff
@@ -1,5 +1,17 @@
 # git-bug rich web UI
 
+## Prerequisites
+
+<div align="center">
+<img height="119px" alt="ReactJS" src="https://www.pinclipart.com/picdir/big/537-5374089_react-js-logo-clipart.png">
+<img width="150px" src="https://material-ui.com/static/logo_raw.svg">
```

Comment:
Can you download and commit those images (with a simple name) so that the repo stay self-contained?

Reply:
For sure!